content/en/ecosystem/vendors.md

```diff
@@ -4,54 +4,40 @@ description: Vendors who natively support OpenTelemetry
-| SigNoz                     | Yes               | Yes         | [signoz.io](https://signoz.io)
-| SolarWinds                 | Yes               | Yes         | [documentation.solarwinds.com/...](https://documentation.solarwinds.com/en/success_center/observability/default.htm#cshid=third-otel-integration)
-| Splunk                     | Yes               | Yes         | [splunk.com/blog/...](https://www.splunk.com/en_us/blog/conf-splunklive/announcing-native-opentelemetry-support-in-splunk-apm.html)
-| Sumo Logic                 | Yes               | Yes         | [help.sumologic.com/](https://help.sumologic.com/docs/apm/traces/quickstart/)
-| TelemetryHub               | No                | Yes         | [telemetryhub.com](https://app.telemetryhub.com/docs)
-| Teletrace                  | Yes               | Yes         | [docs.teletrace.io](https://docs.teletrace.io/)
-| ThousandEyes (Cisco)       | No                | Yes         | [docs.thousandeyes.com](https://docs.thousandeyes.com/product-documentation/api/opentelemetry)
-| Traceloop                  | No                | Yes         | [traceloop.com](https://www.traceloop.com)
-| Uptrace                    | Yes               | Yes         | [uptrace.dev](https://uptrace.dev)
-
-\* _Vendors are listed alphabetically_.
+A non-exhaustive list of organizations that provide solutions which consume
+OpenTelemetry natively via [OTLP](/docs/specs/otlp/) e.g. observability backends
+or observability pipelines.
+
+Those offerings may be open source, commercial or both. Some organizations
+provide a [distribution](/docs/concepts/distributions/) (a customized version of
+OpenTelemetry components), that provide additional capabilities or ease of use.
+
+{{% ecosystem/vendor-table %}}
+
+To have your organization included,
+[add an entry to the list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/vendors.yaml)
+and submit a PR.
+
+You will need to include[^existing-entries]:
+
+- a link to the documentation on how your offering consumes OpenTelemetry
+  natively via [OTLP](http://localhost:1313/docs/specs/otlp/).
+- if applicable, a link to your distribution.
+- if applicable, a link that proves that your offering is open source. An open
+  source distribution does not qualify your offering to be marked "open source".
+- a point of contact (GitHub handle or email address), we can reach out for
+  future inquiries.
+
+Note, that this list is for organizations that consume OpenTelemetry to offer
+Observability to end users.
+
+If you adopt OpenTelemetry for Observability as an end user and do not provide
+any kind of services around OpenTelemetry, see [Adopters](/ecosystem/adopters).
+<br /> If you provide a library, service or app made observable through
+OpenTelemetry, see [Integrators](/ecosystem/integrations).
+
+[^existing-entries]:
+    Organizations that have been part of this list before July 2023 are exempted
+    from these requirements until January, 1th 2024.
```

Comment:
Please see the copyedits suggested for the corresponding part of the Adopters page, and also make them here.